### PR TITLE
Adds prettier

### PR DESCRIPTION
**Pull request by eamodio**
Prettiers all the things, and as eslint import sorting

### CODE REVIEW

File: src/common/webview.ts [hunk 1]
```diff
@@ -19,7 +19,8 @@ export interface IReplyMessage {
 
 export function getNonce() {
 	let text = '';
-	const possible = 'ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789';
+	const possible =
```
[the remark this answers is not in the document]
Comment (RMacfarlane) [reply src/common/webview.ts:22]: actually it looks like maybe `printWidth` could be changed. could we bump this up to 120?

**Review by RMacfarlane — CHANGES_REQUESTED** [review]

File: src/common/webview.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (eamodio) [reply src/common/webview.ts]: Done

File: src/authentication/configuration.ts [hunk 2]
```diff
@@ -8,30 +8,41 @@ export interface IHostConfiguration {
...
+				'Yes',
+			);
 			if (result === 'Yes') {
 				USE_TEST_SERVER = true;
 				return vscode.Uri.parse(testEnv);
 			}
 		}
 
-		const hostUri: vscode.Uri = host instanceof vscode.Uri ? host : vscode.Uri.parse(host.host);
+		const hostUri: vscode.Uri =
```
[the remark this answers is not in the document]
Comment (eamodio) [reply src/authentication/configuration.ts]: Sadly no, unless the line is too long :(

File: .gitignore-revs [hunk 3]
```diff
@@ -0,0 +1,2 @@
+e8059c539de0fcea91d72a282b9fa56620bf2738
+25d91df6d7639ec02ed50a4b7ebe5a80445c0cd0
```
Comment (RMacfarlane) [inline .gitignore-revs:2]: Did not know excluding things from git-blame was possible, very cool!

**Review by RMacfarlane — APPROVED** [review]
Looks great, thanks for adding this!

**Pull request merged**